design/dedicated-osd-pod.md

```diff
@@ -52,32 +52,25 @@ This change addresses the above limitations in the follow ways:
 
 ### Detailed Device Discovery Process
 
-Each `Device Discovery` Pod walks through device trees to unformatted block devices and stores the device information in `Raw Device Configmap`. 
+Each `Device Discovery` Pod walks through device trees to unformatted block devices and stores the device information in a per node `Raw Device Configmap`. 
 
-A sample of `Raw Device Configmap` is as the following:
+A sample of `Raw Device Configmap` from Node `node1` is as the following:
 
 ```yaml
 apiVersion: v1
 kind: ConfigMap
 metadata:
   namespace: rook-system
-  name: raw-devices
+  name: node1-raw-devices
 data:
-    - node: "node1"
-      devices:
-      -  device-path: /dev/disk/by-id/scsi-dead     # persistent device path
-         size: 214748364800                         # size in byte
-         rotational: 0                              # 0 for ssd/nvme, 1 for hdd, based on reading from sysfs
-         extra: '{ "vendor": "X", "model": "Y" }'   # extra information about the device in json
-      -  device-path: /dev/disk/by-id/scsi-beef     # persistent device path
-         size: 214748364800                         # size in byte
-         rotational: 0                              # 0 for ssd/nvme, 1 for hdd, based on reading from sysfs
-         extra: '{ "vendor": "X", "model": "Y" }'   # extra information about the device in json
-    - node: "node2"
-      devices:
-      -  device-path: /dev/disk/by-partuuid/12345   # persistent device path
-         size: 214748364800                         # size in byte
-         rotational: 1                              # 0 for ssd/nvme, 1 for hdd, based on reading from sysfs
+   devices
+   -  device-path: /dev/disk/by-id/scsi-dead     # persistent device path
+      size: 214748364800                         # size in byte
+      rotational: 0                              # 0 for ssd/nvme, 1 for hdd, based on reading from sysfs
+      extra: '{ "vendor": "X", "model": "Y" }'   # extra information about the device in json
```

Comment:
we should be able to add other properties inline as well. If we decide to add another property such as `vendor`, the discovery pod would discover it and the operator would also need to know how to interpret it from the filter in the crd

Reply:
having a json string instead of adding more types here makes it flexible across platforms. Such information may or may not be as important as size and rotational. 

`vendor` and `model` are examples of what additional information about the device we should retrieve from sysfs. There are many more there such scsi target, revision level, vpd etc, they can also change between kernels.